src/main/java/com/sublinks/sublinksapi/api/lemmy/v3/user/repositories/CaptchaRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.sublinks.sublinksapi.api.lemmy.v3.user.repositories;
+
+import com.sublinks.sublinksapi.api.lemmy.v3.user.dto.Captcha;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+public interface CaptchaRepository extends JpaRepository<Captcha, Long> {
+
+  @Query(value = "select * from captcha where locked is false limit 1", nativeQuery = true)
+  Captcha findFirst();
+
+  @Query(value = "select count(*) from captcha where locked is false", nativeQuery = true)
+  long count();
```

Comment:
here you can just do:

long countByLockedFalse 

OR if i am wrong try:

long countByLocked(Bool locked)

( I am sorry if iam vague here but i dont have an IDE setup currently! )